src/plugin/index.js

```diff
@@ -19,7 +18,10 @@ function Jarvis(options = {}) {
         1337
       : options.port,
 
-    host: "host" in options ? options.host : "localhost"
+    host: "host" in options ? options.host : "localhost",
+
+    packageJsonPath: fs.existsSync(options.packageJsonPath) //Check if path actually exists otherwise fallback to currenty
```

Comment:
We should probably show a console warning if the configured path doesn't exist and we have to use a default. Saving the user some frustration. :+1: